cowbird/services/impl/geoserver.py

```diff
@@ -1,22 +1,27 @@
-from cowbird.request_queue import RequestQueue
+from celery import chain, shared_task
+
+from cowbird.request_task import RequestTask
 from cowbird.services.service import Service
+from cowbird.services.service_factory import ServiceFactory
 
 
 class Geoserver(Service):
     """
-    Keep Geoserver internal representation in synch with the platform.
+    Keep Geoserver internal representation in sync with the platform.
     """
 
     def __init__(self, name, url):
         super(Geoserver, self).__init__(name, url)
-        self.req_queue = RequestQueue()
 
     def get_resource_id(self, resource_full_name):
         # type (str) -> str
         raise NotImplementedError
 
     def user_created(self, user_name):
-        raise NotImplementedError
+        # ..todo: Replace this simple stub with real implementation
+        res = chain(create_workspace.s(user_name),
+                    create_datastore.s("default"))
```

Comment:
use `.signature` instead for better readability?

Reply:
in fact the premitive is not the same...
`def s(self, *args, **kwargs)`
vs
`def signature(self, args=None, *starargs, **starkwargs)`

So while a `basic_task.s(1)` is still called like `basic_task.signature(1)` a `sum_task.s(1, 2)` must then be called like this `sum_task.signature((1, 2))`

I find it more disturbing that the `.s` which is quiet common when your used to use celery

Reply:
Fair enough. If not a direct replacement, `s` is fine.